TaskList/app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1 @@
+<%= render partial: "form_partial", locals: {action: 'update' , method_type: :patch} %>
```

Comment:
Great use of a partial with local variables for the form action! Where is the `method_type` variable being used?